2024-03-28 00:09:07,312 INFO    StreamThr :2576 [internal.py:wandb_internal():86] W&B internal server running at pid: 2576, started at: 2024-03-28 00:09:07.305609
2024-03-28 00:09:07,320 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status
2024-03-28 00:09:07,348 INFO    WriterThread:2576 [datastore.py:open_for_write():87] open: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\run-thohd74n.wandb
2024-03-28 00:09:07,353 DEBUG   SenderThread:2576 [sender.py:send():378] send: header
2024-03-28 00:09:07,361 DEBUG   SenderThread:2576 [sender.py:send():378] send: run
2024-03-28 00:09:08,811 INFO    SenderThread:2576 [dir_watcher.py:__init__():211] watching files in: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files
2024-03-28 00:09:08,811 INFO    SenderThread:2576 [sender.py:_start_run_threads():1099] run started: thohd74n with start time 1711564747.317422
2024-03-28 00:09:08,881 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: check_version
2024-03-28 00:09:08,884 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: check_version
2024-03-28 00:09:10,294 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: run_start
2024-03-28 00:09:11,175 DEBUG   HandlerThread:2576 [system_info.py:__init__():26] System info init
2024-03-28 00:09:11,175 DEBUG   HandlerThread:2576 [system_info.py:__init__():41] System info init done
2024-03-28 00:09:11,175 INFO    HandlerThread:2576 [system_monitor.py:start():194] Starting system monitor
2024-03-28 00:09:11,175 INFO    SystemMonitor:2576 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-03-28 00:09:11,175 INFO    HandlerThread:2576 [system_monitor.py:probe():214] Collecting system info
2024-03-28 00:09:11,182 INFO    SystemMonitor:2576 [interfaces.py:start():190] Started cpu monitoring
2024-03-28 00:09:11,182 INFO    SystemMonitor:2576 [interfaces.py:start():190] Started disk monitoring
2024-03-28 00:09:11,182 INFO    SystemMonitor:2576 [interfaces.py:start():190] Started gpu monitoring
2024-03-28 00:09:11,182 INFO    SystemMonitor:2576 [interfaces.py:start():190] Started memory monitoring
2024-03-28 00:09:11,191 INFO    SystemMonitor:2576 [interfaces.py:start():190] Started network monitoring
2024-03-28 00:09:11,203 DEBUG   HandlerThread:2576 [system_info.py:probe():150] Probing system
2024-03-28 00:09:11,203 DEBUG   HandlerThread:2576 [gitlib.py:_init_repo():56] git repository is invalid
2024-03-28 00:09:11,203 DEBUG   HandlerThread:2576 [system_info.py:probe():198] Probing system done
2024-03-28 00:09:11,210 DEBUG   HandlerThread:2576 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22631-SP0', 'python': '3.9.13', 'heartbeatAt': '2024-03-27T18:39:11.203404', 'startedAt': '2024-03-27T18:39:07.262289', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'C:\\Users\\chinm\\Desktop\\FL-FINAL\\CIFAR\\server.py', 'codePathLocal': 'server.py', 'codePath': 'server.py', 'host': 'MSI', 'username': 'chinm', 'executable': 'C:\\Users\\chinm\\anaconda3\\New folder\\python.exe', 'cpu_count': 6, 'cpu_count_logical': 12, 'cpu_freq': {'current': 2496.0, 'min': 0.0, 'max': 2496.0}, 'cpu_freq_per_core': [{'current': 2496.0, 'min': 0.0, 'max': 2496.0}], 'disk': {'/': {'total': 458.81444931030273, 'used': 429.0327262878418}}, 'gpu': 'NVIDIA GeForce RTX 3050 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3050 Laptop GPU', 'memory_total': 4294967296}], 'memory': {'total': 7.8259735107421875}}
2024-03-28 00:09:11,210 INFO    HandlerThread:2576 [system_monitor.py:probe():224] Finished collecting system info
2024-03-28 00:09:11,210 INFO    HandlerThread:2576 [system_monitor.py:probe():227] Publishing system info
2024-03-28 00:09:11,210 DEBUG   HandlerThread:2576 [system_info.py:_save_conda():207] Saving list of conda packages installed into the current environment
2024-03-28 00:09:11,220 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:09:11,825 INFO    Thread-16 :2576 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\conda-environment.yaml
2024-03-28 00:09:13,835 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:13,835 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:16,461 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:16,461 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:19,101 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:19,101 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:21,746 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:21,747 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:24,391 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:24,394 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:27,036 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:27,036 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:29,688 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:29,688 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:31,817 DEBUG   HandlerThread:2576 [system_info.py:_save_conda():219] Saving conda packages done
2024-03-28 00:09:31,817 INFO    HandlerThread:2576 [system_monitor.py:probe():229] Finished publishing system info
2024-03-28 00:09:32,011 INFO    Thread-16 :2576 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\conda-environment.yaml
2024-03-28 00:09:32,011 INFO    Thread-16 :2576 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\wandb-metadata.json
2024-03-28 00:09:32,051 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:09:32,051 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: keepalive
2024-03-28 00:09:32,051 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:09:32,051 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: keepalive
2024-03-28 00:09:32,051 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:09:32,051 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: keepalive
2024-03-28 00:09:32,051 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:09:32,051 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: keepalive
2024-03-28 00:09:32,051 DEBUG   SenderThread:2576 [sender.py:send():378] send: files
2024-03-28 00:09:32,056 INFO    SenderThread:2576 [sender.py:_save_file():1365] saving file wandb-metadata.json with policy now
2024-03-28 00:09:32,341 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:32,341 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:34,070 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:09:34,221 INFO    wandb-upload_0:2576 [upload_job.py:push():131] Uploaded file C:\Users\chinm\AppData\Local\Temp\tmpnjnr2ueywandb\pbc9nc5t-wandb-metadata.json
2024-03-28 00:09:34,984 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:34,984 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:35,381 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: python_packages
2024-03-28 00:09:35,381 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: python_packages
2024-03-28 00:09:35,381 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:09:35,381 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:09:35,381 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:09:35,751 DEBUG   SenderThread:2576 [sender.py:send():378] send: telemetry
2024-03-28 00:09:36,051 INFO    Thread-16 :2576 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\output.log
2024-03-28 00:09:36,051 INFO    Thread-16 :2576 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\requirements.txt
2024-03-28 00:09:37,602 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:37,602 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:38,072 INFO    Thread-16 :2576 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\output.log
2024-03-28 00:09:39,611 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:09:39,796 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:09:41,114 INFO    Thread-16 :2576 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\config.yaml
2024-03-28 00:09:42,241 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:42,241 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:44,871 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:44,871 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:45,313 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:09:47,495 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:47,495 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:50,161 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:50,161 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:50,182 INFO    Thread-16 :2576 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\output.log
2024-03-28 00:09:50,343 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:09:50,391 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:09:50,391 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:09:50,391 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:09:52,806 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:52,806 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:55,451 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:55,451 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:55,881 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:09:58,102 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:09:58,102 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:00,732 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:00,732 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:00,925 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:03,361 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:03,361 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:05,404 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:10:05,406 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:10:05,408 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:10:06,002 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:06,002 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:06,826 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:08,631 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:08,631 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:11,201 DEBUG   SystemMonitor:2576 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-03-28 00:10:11,201 DEBUG   SenderThread:2576 [sender.py:send():378] send: stats
2024-03-28 00:10:11,251 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:11,251 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:12,218 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:13,291 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:13,291 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:15,933 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:15,936 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:16,428 INFO    Thread-16 :2576 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\output.log
2024-03-28 00:10:17,952 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:10:18,223 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:19,981 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:19,981 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:20,409 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:10:20,410 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:10:20,410 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:10:22,624 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:22,624 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:23,832 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:25,281 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:25,281 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:27,981 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:27,981 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:28,866 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:30,622 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:30,622 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:33,302 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:33,302 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:33,916 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:35,406 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:10:35,406 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:10:35,422 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:10:35,966 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:35,967 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:38,645 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:38,650 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:39,300 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:41,211 DEBUG   SenderThread:2576 [sender.py:send():378] send: stats
2024-03-28 00:10:41,362 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:41,364 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:44,083 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:44,083 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:45,246 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:46,820 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:46,820 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:49,510 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:49,510 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:50,310 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:50,419 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:10:50,421 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:10:50,433 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:10:52,220 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:52,220 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:54,285 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:54,290 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:56,234 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:10:57,065 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:10:59,745 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:10:59,746 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:01,275 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:02,452 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:02,456 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:05,175 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:05,175 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:05,424 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:11:05,440 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:11:05,440 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:11:06,986 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:08,096 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:11:10,790 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:10,790 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:11,220 DEBUG   SenderThread:2576 [sender.py:send():378] send: stats
2024-03-28 00:11:12,238 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:13,475 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:13,475 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:16,213 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:16,220 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:17,277 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:18,930 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:18,930 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:20,446 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:11:20,448 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:11:20,451 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:11:21,737 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:11:22,880 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:24,503 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:11:27,320 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:11:27,965 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:30,018 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:30,019 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:32,855 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:11:33,038 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:35,443 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:11:35,457 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:11:35,457 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:11:35,620 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:11:38,354 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:38,354 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:39,390 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:41,143 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:11:41,233 DEBUG   SenderThread:2576 [sender.py:send():378] send: stats
2024-03-28 00:11:44,670 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:11:44,970 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:47,388 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:47,388 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:50,577 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:50,586 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:11:50,586 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:11:50,664 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:11:50,859 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:11:53,849 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:11:56,434 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:11:56,570 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:56,572 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:11:59,350 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:01,466 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:02,112 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:04,900 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:05,440 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:12:05,484 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:12:05,490 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:12:06,961 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:07,686 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:10,420 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:10,420 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:11,234 DEBUG   SenderThread:2576 [sender.py:send():378] send: stats
2024-03-28 00:12:12,240 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:13,400 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:16,338 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:17,473 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:19,020 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:19,024 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:20,460 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:12:20,490 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:12:20,493 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:12:21,770 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:22,840 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:24,510 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:24,510 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:27,261 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:27,927 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:30,020 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:32,973 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:33,003 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:35,463 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:12:35,520 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:12:35,520 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:12:35,710 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:35,710 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:37,986 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:38,420 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:38,430 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:41,161 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:41,161 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:41,242 DEBUG   SenderThread:2576 [sender.py:send():378] send: stats
2024-03-28 00:12:43,301 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:43,859 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:43,860 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:45,905 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:48,381 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:48,625 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:48,625 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:50,478 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:12:50,545 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:12:50,547 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:12:51,423 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:12:54,062 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:54,210 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:54,216 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:56,960 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:56,961 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:59,124 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:12:59,666 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:12:59,666 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:13:02,350 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:13:02,350 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:13:04,215 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:13:05,095 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-28 00:13:05,097 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:13:05,483 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-28 00:13:05,533 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: stop_status
2024-03-28 00:13:05,533 DEBUG   SenderThread:2576 [sender.py:send_request():405] send_request: stop_status
2024-03-28 00:13:07,882 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:13:10,110 DEBUG   HandlerThread:2576 [handler.py:handle_request():146] handle_request: status_report
2024-03-28 00:13:10,690 ERROR   gpu       :2576 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-28 00:13:11,250 DEBUG   SenderThread:2576 [sender.py:send():378] send: stats
2024-03-28 00:13:12,220 WARNING StreamThr :2576 [internal.py:is_dead():414] Internal process exiting, parent pid 20544 disappeared
2024-03-28 00:13:12,221 ERROR   StreamThr :2576 [internal.py:wandb_internal():152] Internal process shutdown.
2024-03-28 00:13:12,262 INFO    HandlerThread:2576 [handler.py:finish():869] shutting down handler
2024-03-28 00:13:12,262 INFO    SenderThread:2576 [sender.py:finish():1521] shutting down sender
2024-03-28 00:13:12,262 INFO    WriterThread:2576 [datastore.py:close():296] close: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\run-thohd74n.wandb
2024-03-28 00:13:12,262 INFO    HandlerThread:2576 [system_monitor.py:finish():203] Stopping system monitor
2024-03-28 00:13:12,262 INFO    HandlerThread:2576 [interfaces.py:finish():202] Joined cpu monitor
2024-03-28 00:13:12,262 DEBUG   SystemMonitor:2576 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-03-28 00:13:12,262 INFO    HandlerThread:2576 [interfaces.py:finish():202] Joined disk monitor
2024-03-28 00:13:12,270 DEBUG   SystemMonitor:2576 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-03-28 00:13:12,886 INFO    HandlerThread:2576 [interfaces.py:finish():202] Joined gpu monitor
2024-03-28 00:13:12,886 INFO    HandlerThread:2576 [interfaces.py:finish():202] Joined memory monitor
2024-03-28 00:13:12,886 INFO    HandlerThread:2576 [interfaces.py:finish():202] Joined network monitor
2024-03-28 00:13:13,401 INFO    SenderThread:2576 [dir_watcher.py:finish():358] shutting down directory watcher
2024-03-28 00:13:13,440 INFO    Thread-16 :2576 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\output.log
2024-03-28 00:13:13,443 INFO    SenderThread:2576 [dir_watcher.py:finish():388] scan: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files
2024-03-28 00:13:13,444 INFO    SenderThread:2576 [dir_watcher.py:finish():402] scan save: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\conda-environment.yaml conda-environment.yaml
2024-03-28 00:13:13,445 INFO    SenderThread:2576 [dir_watcher.py:finish():402] scan save: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\config.yaml config.yaml
2024-03-28 00:13:13,453 INFO    SenderThread:2576 [dir_watcher.py:finish():402] scan save: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\output.log output.log
2024-03-28 00:13:13,459 INFO    SenderThread:2576 [dir_watcher.py:finish():402] scan save: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\requirements.txt requirements.txt
2024-03-28 00:13:13,464 INFO    SenderThread:2576 [dir_watcher.py:finish():402] scan save: C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\wandb-metadata.json wandb-metadata.json
2024-03-28 00:13:13,464 INFO    SenderThread:2576 [file_pusher.py:finish():172] shutting down file pusher
2024-03-28 00:13:13,464 INFO    SenderThread:2576 [file_pusher.py:join():178] waiting for file pusher
2024-03-28 00:13:14,330 INFO    wandb-upload_0:2576 [upload_job.py:push():131] Uploaded file C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\conda-environment.yaml
2024-03-28 00:13:15,030 INFO    wandb-upload_2:2576 [upload_job.py:push():131] Uploaded file C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\output.log
2024-03-28 00:13:15,463 INFO    wandb-upload_1:2576 [upload_job.py:push():131] Uploaded file C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\config.yaml
2024-03-28 00:13:16,370 INFO    wandb-upload_3:2576 [upload_job.py:push():131] Uploaded file C:\Users\chinm\Desktop\FL-FINAL\CIFAR\wandb\run-20240328_000907-thohd74n\files\requirements.txt
2024-03-28 00:13:16,375 INFO    MainThread:2576 [internal.py:handle_exit():76] Internal process exited
2024-03-28 00:13:17,020 INFO    SenderThread:2576 [file_stream.py:finish():595] file stream finish called
